plugins/polls/app/models/poll_vote.rb

```diff
@@ -0,0 +1,6 @@
+class PollVote < ActiveRecord::Base
+  belongs_to :poll
+  belongs_to :ordergroup
+  belongs_to :user
+  has_many :poll_choices, dependent: :destroy
+end
```

Comment:
I like that the models start with the same name, keeps them together.

Reply:
what do you mean? rename `PollChoice` to `PollVoteChoice` and `poll_choices` to `poll_vote_choices`?

Reply:
It's good as it is, that's what I was trying to say :+1: